### DISCUSSION TITLE
Allow the user to select whether column headers are shown

### DISCUSSION THREAD

**Discussion by gmancoder** [orchidsoftware/platform#2905] [Ideas]
I see in the Table class, the determinant for whether headers are shown is if there are 2 or greater columns. Is there a way to gracefully force them to appear on a single column table (or not appear when >2)?

**Comment by tabuna** [comment]
You can create a table class using the `artisan orchid:table` command, and then override the following method in your class:

```php
protected function hasHeader(Collection $columns, Collection|Paginator|CursorPaginator $row): bool
{
    return true;
}
```

This method is responsible for displaying the table headers.

**Reply by gmancoder** [reply]
Thanks so much @tabuna
